src/variant17.lib/variant17/Variant.h

```diff
@@ -207,12 +220,7 @@ struct Variant {
         if (whichValue != npos) destruct();
         whichValue = npos;
         constructOf(type<T>, std::forward<Args>(args)...);
-        whichValue = whichOf<T>();
-    }
-
-    template<class T>
-    constexpr static auto whichOf(Type<T> = {}) -> Which {

```

Comment:
I guess it's better to leave a `whichOf` around that simply calls `Which::of` but keeps the old API.

Most users of a `Variant` use it to store values and will not use the `Which`-type except for checking the content.